### DISCUSSION TITLE
compatible with Spring boot 3.x

### DISCUSSION THREAD

**Discussion by flinden68** [ExpediaGroup/graphql-kotlin#1834] [Q&A] [answered]
In the release 6.5.3 I noticed that the spring boot version 2.7.2
Are there plans to make this project compatible with Spring Boot 3.x??

**Comment by dariuszkuc** [comment, marked as answer]
Version 7 is targeting Spring Boot 3.x. I believe you should be able to make v6 work with Spring Boot 3 but that might require some additional local changes (see https://github.com/ExpediaGroup/graphql-kotlin/pull/1638).
